Formula/n/netatalk.rb

```diff
@@ -12,8 +10,16 @@ class Netatalk < Formula
     "BSD-3-Clause",
     "MIT",
   ]
+  revision 1
   head "https://github.com/Netatalk/netatalk.git", branch: "main"
 
+  stable do
+    url "https://github.com/Netatalk/netatalk/releases/download/netatalk-4-3-0/netatalk-4.3.0.tar.xz"
+    sha256 "bc71a6a2f11cf00cb69ef13e8487ab78d1241ae535feb010220c74c89dc890fb"
+
+    depends_on "berkeley-db@5" # macOS bdb library lacks DBC type etc.
```

Comment:
Is the idea to keep this here indefinitely, or to replace bdb with sqlite at some point?

If it's the latter, will that break existing users?

Reply:
I was thinking more in terms of the formula itself -- if you've decided to drop BDB, and that might break users, that's fine, we can assume you've weighed this decision appropriately.

What I want to avoid is premature removal of the BDB backend in the formula that results in user breakage, when you/upstream haven't done that yet.

Reply:
Sounds good!